### PR TITLE
DataGrid/TreeList: add Accessibility topic

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/05 UI Components/DataGrid/05 Accessibility/05 Accessibility Features Overview.md
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/05 Accessibility Features Overview.md:1]: Let's write support level without icons. Instead, write in words (Supports, Supports with exceptions)

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        <th>Section 508 criteria</th>
+        <th>WCAG 2.x criteria</th>
+        <th>Exception description</th>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>1.1.1 Non-text Content (Level A)</td>
+        <td>
+            <ul>
+                <li>HeaderFilter icons do not have <code>alt</code> attributes.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                <li>Filter icons in the filter row do not have aria-label attributes.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        </td>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>1.3.1 Info and Relationships (Level A)</td>
+        <td>
+            <ul>
+                <li>Delete confirmation messages are not pronounced.</li>
+                <li>The column fixing feature does not meet this criterion.</li>
+                <li>Summaries are not pronounced.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: Move it to 2.1.1 Keyboard (Level A) item. And replace description on 'Group footer summary and total summary are not supported'

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+                <li>FilterRow may lose the <code>aria-describedby</code> attribute.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: it is not reproduced

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        <td>1.3.2 Meaningful Sequence (Level A) <br> 2.4.3 Focus Order (Level A)</td>
+        <td>The column fixing feature in DataGrid does not meet this criterion.</td>
+    </tr>
+    <tr>
+        <td>302.3 Without Perception of Color</td>
+        <td>1.4.1 Use of Color (Level A)</td>
+        <td>DataGrid has only visual means of conveying information for the following elements:
+            <ul>
+                <li>Modified cells and deleted rows in batch edit mode.</li>
+                <li>Current page size and index.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+        <td>1.3.2 Meaningful Sequence (Level A) <br> 2.4.3 Focus Order (Level A)</td>
+        <td>The column fixing feature in TreeList does not meet this criterion.</td>
+    </tr>
+    <tr>
+        <td>302.3 Without Perception of Color</td>
+        <td>1.4.1 Use of Color (Level A)</td>
+        <td>TreeList has only visual means of conveying information for the following elements:
+            <ul>
+                <li>Modified cells and deleted rows in batch edit mode.</li>
+                <li>Current page size and index.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+        <th>Section 508 criteria</th>
+        <th>WCAG 2.x criteria</th>
+        <th>Exception description</th>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>1.1.1 Non-text Content (Level A)</td>
+        <td>
+            <ul>
+                <li>HeaderFilter icons do not have <code>alt</code> attributes.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+                <li>Filter icons in the filter row do not have aria-label attributes.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+        </td>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>1.3.1 Info and Relationships (Level A)</td>
+        <td>
+            <ul>
+                <li>Delete confirmation messages are not pronounced.</li>
+                <li>The column fixing feature does not meet this criterion.</li>
+                <li>Summaries are not pronounced.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: TreeList is not support the summary

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+                <li>FilterRow may lose the <code>aria-describedby</code> attribute.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: it is not reproduced

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+                <li>Cells in invalid state.</li>
+            </ul>    
+        </td>
+    </tr>
+    <tr>
+        <td>11.5.2.12 Execution of available actions</td>
+        <td>2.1.1 Keyboard (Level A)</td>
+        <td><ul>
+                <li>Keyboard navigation does not work properly in <code>repaintChangesOnly</code> mode.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not reproduced

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+                <li>Current page size and index.</li>
+                <li>Cells in invalid state.</li>
+            </ul>    
+        </td>
+    </tr>
+    <tr>
+        <td>11.5.2.12 Execution of available actions</td>
+        <td>2.1.1 Keyboard (Level A)</td>
+        <td><ul>
+                <li>Keyboard navigation does not work properly in <code>repaintChangesOnly</code> mode.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not reproduced

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+                <li>The column fixing and column visibility customization features cause accessibility issues.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: Issue with column fixing is described already. Leave only description about Column chooser in 'drag' mode.

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 14]
```diff
@@ -0,0 +1,112 @@
...
+                <li>The column fixing and column visibility customization features cause accessibility issues.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: Issue with column fixing is described already. Leave only description about Column chooser in 'drag' mode.

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 15]
```diff
@@ -0,0 +1,112 @@
...
+                <li>Users cannot export data.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 16]
```diff
@@ -0,0 +1,118 @@
...
+                <li>Users cannot export data.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 17]
```diff
@@ -0,0 +1,112 @@
...
+                <li>The master-detail feature does not support keyboard navigation.</li>
+            </ul>    
+        </td>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>2.1.2 No Keyboard Trap (Level A)</td>
+        <td><ul>
+            <li>Users must use the 'Shift+Tab' shortcut to move focus from the Column Chooser in DataGrid.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 18]
```diff
@@ -0,0 +1,118 @@
...
+                <li>The master-detail feature does not support keyboard navigation.</li>
+            </ul>    
+        </td>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>2.1.2 No Keyboard Trap (Level A)</td>
+        <td><ul>
+            <li>Users must use the 'Shift+Tab' shortcut to move focus from the Column Chooser in TreeList.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 19]
```diff
@@ -0,0 +1,118 @@
...
+            <li>The column fixing feature in TreeList might cause a keyboard trap.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not reproduced

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 20]
```diff
@@ -0,0 +1,112 @@
...
+            <li>The column fixing feature in DataGrid might cause a keyboard trap.</li>
+        </ul></td>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: Keyboard trap appears when there is a group footer summary

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 21]
```diff
@@ -0,0 +1,112 @@
...
+                <li>The master-detail feature does not support keyboard navigation.</li>
+            </ul>    
+        </td>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>2.1.2 No Keyboard Trap (Level A)</td>
+        <td><ul>
+            <li>Users must use the 'Shift+Tab' shortcut to move focus from the Column Chooser in DataGrid.</li>
+            <li>The column fixing feature in DataGrid might cause a keyboard trap.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not reproduced

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 22]
```diff
@@ -0,0 +1,112 @@
...
+        </ul></td>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>2.1.4 Character Key Shortcuts (Level A 2.1 only)</td>
+        <td>Adaptive rows cannot be expanded/collapsed using keyboard navigation.</td>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: Move it to '2.1.1 Keyboard (Level A)' item

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 23]
```diff
@@ -0,0 +1,118 @@
...
+        </ul></td>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>2.1.4 Character Key Shortcuts (Level A 2.1 only)</td>
+        <td>Adaptive rows cannot be expanded/collapsed using keyboard navigation.</td>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: Move it to '2.1.1 Keyboard (Level A)' item

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 24]
```diff
@@ -0,0 +1,112 @@
...
+        <td>3.3.1 Error Identification (Level A) <br> 3.3.3 Error Suggestion (Level AA) <br> 4.1.2 Name, Role, Value (Level A)</td>
+        <td>The validation state and error message of DataGrid are not pronounced when users focus the control or edit data.</td>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>3.3.2 Labels or Instructions (Level A)</td>
+        <td>
+            <ul>
+                <li>Cells in batch edit mode do not supply any instructions.</li>
+                <li>The sorting and export buttons, HeaderFilter, and FilterRow do not supply any instructions.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 25]
```diff
@@ -0,0 +1,118 @@
...
+        <td>3.3.1 Error Identification (Level A) <br> 3.3.3 Error Suggestion (Level AA) <br> 4.1.2 Name, Role, Value (Level A)</td>
+        <td>The validation state and error message of TreeList are not pronounced when users focus the control or edit data.</td>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>3.3.2 Labels or Instructions (Level A)</td>
+        <td>
+            <ul>
+                <li>Cells in batch edit mode do not supply any instructions.</li>
+                <li>The sorting and export buttons, HeaderFilter, and FilterRow do not supply any instructions.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 26]
```diff
@@ -0,0 +1,112 @@
...
+                <li>The ‘Select All’ checkbox does not supply instructions.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 27]
```diff
@@ -0,0 +1,118 @@
...
+                <li>The ‘Select All’ checkbox does not supply instructions.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 28]
```diff
@@ -0,0 +1,118 @@
...
+    <tr>
+        <td>502.3.7 Hierarchical Relationships</td>
+        <td>-</td>
+        <td>TreeList only conveys information about the nesting level (aria-level).</td>
+    </tr>
+    <tr>
+        <td>502.3.14 Event Notification <br> 11.5.2.15 Change notification</td>
+        <td>-</td>
+        <td><ul>
+                <li>TreeView in ColumnChooser and HeaderFilter applies redundant aria-expanded attributes to these elements.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 29]
```diff
@@ -0,0 +1,112 @@
...
+    <tr>
+        <td>502.3.3 Row, Column, and Headers</td>
+        <td>-</td>
+        <td>DataGrid does not convey information about headers if the column fixing feature is enabled.</td>
+    </tr>
+    <tr>
+        <td>502.3.14 Event Notification <br> 11.5.2.15 Change notification</td>
+        <td>-</td>
+        <td><ul>
+                <li>TreeView in ColumnChooser and HeaderFilter applies redundant aria-expanded attributes to these elements.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 30]
```diff
@@ -0,0 +1,118 @@
...
+                <li>Validation frame, sort indexes, and the export button in the toolbar are not accessible.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: Issue with export button is not actual

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 31]
```diff
@@ -0,0 +1,112 @@
...
+                <li>Validation frame, sort indexes, and the export button in the toolbar are not accessible.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: Issue with export button is not actual

Comment (Alyar666) [reply concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: And replace 'indexes' -> 'indices'

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md (hunk 30 above)
Comment (Alyar666) [reply concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: And replace 'indexes' -> 'indices'

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 32]
```diff
@@ -0,0 +1,112 @@
...
+                <li>Delete confirmation messages are not pronounced.</li>
+                <li>The column fixing feature does not meet this criterion.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 33]
```diff
@@ -0,0 +1,118 @@
...
+                <li>Delete confirmation messages are not pronounced.</li>
+                <li>The column fixing feature does not meet this criterion.</li>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: It is not actual

File: concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 34]
```diff
@@ -0,0 +1,118 @@
...
+                <li>TreeList does not convey information about expandable/collapsible nodes.</li>
+            </ul>    
+        </td>
+    </tr>
+    <tr>
+        <td>11.5.2.13 Tracking of focus and selection attributes</td>
```
Comment (Alyar666) [inline concepts/05 UI Components/TreeList/05 Accessibility/10 Accessibility Standards Compliance.md]: You can remove all row

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md [hunk 35]
```diff
@@ -0,0 +1,112 @@
...
+            </ul>    
+        </td>
+    </tr>
+    <tr>
+        <td>11.5.2.13 Tracking of focus and selection attributes</td>
```
Comment (Alyar666) [inline concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: You can remove all row

**Review by Alyar666 — CHANGES_REQUESTED** [review]

File: concepts/05 UI Components/DataGrid/05 Accessibility/05 Accessibility Features Overview.md
Comment (vladaskorohodova) [reply concepts/05 UI Components/DataGrid/05 Accessibility/05 Accessibility Features Overview.md:1]: I can't do that, the table with icons is approved by PMs and will be in every accessibility topic

File: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md (hunk 14 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: What about ColumnChooser in 'drag' mode? What exactly does not work?

Comment (Alyar666) [reply concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md]: > What about ColumnChooser in 'drag' mode? What exactly does not work?

 Keyboard navigation not working for column chooser items

File: concepts/05 UI Components/DataGrid/50 Selection/10 User Interaction.md [hunk 36]
```diff
@@ -273,7 +273,7 @@ You can prevent users from selecting all rows by setting the **selection**.[allo
     
 ---
 
-The [showCheckBoxesMode](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/selection/showCheckBoxesMode.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#showCheckBoxesMode') property specifies when the UI component renders check boxes in the selection column. For example, the following code tells the UI component to never render them, though a user can still select rows using [keyboard shortcuts](/concepts/05%20UI%20Components/DataGrid/75%20Keyboard%20Support.md '/Documentation/Guide/UI_Components/DataGrid/Keyboard_Support/'):
+The [showCheckBoxesMode](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/selection/showCheckBoxesMode.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#showCheckBoxesMode') property specifies when the UI component renders check boxes in the selection column. For example, the following code tells the UI component to never render them, though a user can still select rows using [keyboard shortcuts](/Documentation/Guide/UI_Components/DataGrid/Accessibility/#Keyboard_Navigation):
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/DataGrid/50 Selection/10 User Interaction.md]:
```suggestion
The [showCheckBoxesMode](/api-reference/10%20UI%20Components/dxDataGrid/1%20Configuration/selection/showCheckBoxesMode.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/selection/#showCheckBoxesMode') the property specifies when the UI component renders checkboxes in the selection column. For example, the following code tells the UI component to never render them, though a user can still select rows using [keyboard shortcuts](/Documentation/Guide/UI_Components/DataGrid/Accessibility/#Keyboard_Navigation):
```

File: concepts/05 UI Components/TreeList/05 Accessibility/15 Keyboard Navigation.md [hunk 37]
```diff
@@ -1,4 +1,4 @@
-A user can interact with the UI component using the following keys:
+An end user can use the following keys to interact with the TreeList component:
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/TreeList/05 Accessibility/15 Keyboard Navigation.md]: Can you explain why you change it from "A user" to "An end user"?

```suggestion
A user can use the following keys to interact with the TreeList component: 
```

**Review by dirk-pieterse — APPROVED** [review]

File: concepts/05 UI Components/TreeList/05 Accessibility/15 Keyboard Navigation.md (hunk 37 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/TreeList/05 Accessibility/15 Keyboard Navigation.md]: I felt like I had to accentuate here that exactly an end user can use keyboard navigation.

Comment (dirk-pieterse) [reply concepts/05 UI Components/TreeList/05 Accessibility/15 Keyboard Navigation.md]: The distinction is between the reader and the user - when we say 'user', it is about the user of the software or application, not the person reading the doc or instructions

**Review by Alyar666 — APPROVED** [review]

File: concepts/05 UI Components/TreeList/05 Accessibility/15 Keyboard Navigation.md (hunk 37 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/TreeList/05 Accessibility/15 Keyboard Navigation.md]: Okay, I will go with 'user' then.

**Review by Raushen — APPROVED** [review]

**Pull request merged**